recipes/sqcircuit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "SQcircuit" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9627fb5bdce35018559beb5153ba6c17d6e4529c6a4152b1c07665e1e072d21f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - dill >=0.3.4
+    - matplotlib >=3.0.0
+    - numpy >=1.14.2
+    - pip
+    - python
+    - qutip >=4.3.1
+    - scipy >=1.1.0
+  run:
+    - dill >=0.3.4
+    - matplotlib >=3.0.0
+    - numpy >=1.14.2
+    - python
+    - qutip >=4.3.1
+    - scipy >=1.1.0
+
+test:
+  imports:
+    - SQcircuit
+
+about:
+  home: "https://github.com/stanfordLINQS/SQcircuit"
```

Comment:
This is currently 404-ing, because I assume this repo is private. Is that going to change?

Reply:
Yes the repo is private, and we are going to change it in few days. Since we want to publish a paper. I want to make sure that every thing is on track. It is going to be public in few days.

Reply:
That's fine.

Reply:
You can also remove @safavilinqs from maintainer for now.